src/caret_analyze/plot/visualize_lib/bokeh/timeseries.py

```diff
@@ -66,20 +67,51 @@ def create_figure(self) -> Figure:
         # Apply xaxis offset
         records_range = Range([to.to_records() for to in target_objects])
         frame_min, frame_max = records_range.get_range()
-        if self._xaxis_type == 'system_time':
+        converter: Optional[ClockConverter] = None
+        if self._xaxis_type == 'sim_time':
+            # TODO: refactor
+            # get converter
+            if isinstance(target_objects[0], Communication):
+                for comm in target_objects:
+                    assert isinstance(comm, Communication)
+                    if comm.callback_subscription:
+                        converter_cb = comm.callback_subscription
+                        converter = converter_cb._provider.get_sim_time_converter()
+                        break
+            else:
+                converter = target_objects[0]._provider.get_sim_time_converter()
+        if converter:
+            frame_min_convert = converter.convert(frame_min)
+            frame_max_convert = converter.convert(frame_max)
+            x_range_name = 'x_plot_axis'
+            apply_x_axis_offset(fig, frame_min_convert, frame_max_convert, x_range_name)
+        elif self._xaxis_type == 'system_time':
             apply_x_axis_offset(fig, frame_min, frame_max)
 
         # Draw lines
         color_selector = ColorSelectorFactory.create_instance(coloring_rule='unique')
         legend_manager = LegendManager()
-        line_source = LineSource(legend_manager, target_objects[0], frame_min, self._xaxis_type)
+        if converter:
+            line_source = \
+                LineSource(legend_manager, target_objects[0], frame_min_convert, self._xaxis_type)
+        else:
+            line_source = \
+                LineSource(legend_manager, target_objects[0], frame_min, self._xaxis_type)
         fig.add_tools(line_source.create_hover())
         for to, timeseries in zip(target_objects, timeseries_records_list):
-            renderer = fig.line(
-                'x', 'y',
-                source=line_source.generate(to, timeseries),
-                color=color_selector.get_color()
-            )
+            if converter:
+                renderer = fig.line(
+                    'x', 'y',
+                    source=line_source.generate(to, timeseries),
+                    color=color_selector.get_color(),
+                    x_range_name=x_range_name
+                )
+            else:
+                renderer = fig.line(
+                    'x', 'y',
+                    source=line_source.generate(to, timeseries),
+                    color=color_selector.get_color()
+                )
```

Comment:
```suggestion
            renderer = fig.line(
                'x', 'y',
                source=line_source.generate(to, timeseries),
                color=color_selector.get_color(),
                x_range_name=x_range_name
            )
```

Reply:
Reflecting the changes mentioned above, result of the test.ipynb file is below.
`ValueError: failed to validate GlyphRenderer(id='1047', ...).x_range_name: expected a value of type str, got None of type NoneType`

Reply:
@tamegaictc2 
It appears that x_range_name is a specification that does not accept None.

Only the part of `renderer = ...` should be implemented with if else.

Reply:
From the difference between the return values of `records_range.get_range()` and `converter.convert()`, it is better to have another variable for `flame_min/max` and `flame_min/max_contert`.

Hence, there is no need to apply the above suggestion.